ports/rp2/pendsv.h

```diff
@@ -42,6 +42,9 @@ enum {
 
 #define PENDSV_DISPATCH_NUM_SLOTS PENDSV_DISPATCH_MAX
 
+// Most other ARM SDKs use macro IRQ_PRI_PENDSV for PendSV IRQ priority
```

Comment:
Most other ARM SDKs?  I would say "other ARM-based ports".

But actually I think the comment should be: This is the IRQ priority that PendSV runs at, to run system-level tasks that preempt the main thread.

Reply:
Oh, my mistake - I actually thought this was a pseudo-CMSIS name not a MicroPython name.